addon/components/frost-modal-binding.js

```diff
@@ -31,6 +52,9 @@ const FrostModalBinding = Component.extend(PropTypesMixin, {
     noBlur: PropTypes.bool,
     targetOutlet: PropTypes.string,
 
+    // State
+    forceDisabledConfirm: PropTypes.bool, // only used when we are currently overriding the confirm button state
```

Comment:
I don't understand why this is part of the interface. It's late though... may just be my brain.

Reply:
`forceDisabledConfirm` was not intended for the public interface, but because we want to initialize it in `getDefaultProps()`, our convention is to declare the state variables in PropTypes as well.  That's not really necessary, just a habit from the other apps I work with.

That said, it is also possible to fiddle that field ourself when we are opting to handle all of the `onConfirm`/`onClose` interactions ourself (by setting `closeOnConfirm=false`).  I'm not sure if that means we should declare it in the props interfaces, or leave that as a "private" flag which users can use at their own risk.

Reply:
ok